src/transformers/trainer.py

```diff
@@ -156,10 +158,15 @@
 if is_apex_available():
     from apex import amp
 
+if is_ipex_available() and "--use_ipex" in sys.argv:
+    import intel_extension_for_pytorch as ipex
+
 if version.parse(torch.__version__) >= version.parse("1.6"):
     _is_torch_generator_available = True
-    _is_native_amp_available = True
-    from torch.cuda.amp import autocast
```

Comment:
Why is this import removed?

Reply:
Because there are two kinds of autocast, for CPU and CUDA. Since torch is imported, we directly use `torch.cpu.amp.autocast` and `torch.cuda.amp.autocast` in contextmgr.

Reply:
Thanks for explaining!